packages/vaadin-upload/src/vaadin-upload.js

```diff
@@ -408,9 +408,16 @@ class UploadElement extends ElementMixin(ThemableMixin(PolymerElement)) {
                 unknown: 'unknown remaining time'
               },
               error: {
-                serverUnavailable: 'Server Unavailable',
-                unexpectedServerError: 'Unexpected Server Error',
-                forbidden: 'Forbidden'
+                serverError: {
```

Comment:
This would definitely be a breaking change. While it's good that you tried to improve on the level of detail, that was not part of the original request, which was just about making the existing messages more user friendly. See also the linked Google docs in the ticket, which gives some background why the ticket was created.

I would suggest to roll back the changes and find more helpful texts for the 3 error messages that already exist. Another thing to consider is that more detailed messages like `File couldn't be uploaded becase service is not availble.` are not necessarily more helpful for an end-user, who has no idea what a service is or how to make it available. Another thing to keep in mind is to try to provide some actionable feedback like `Please try again later`. I would also encourage you to run your texts though a grammar / spell checker before opening a PR, there are quite a few issues with the proposed texts. After doing that it's of course always OK to ask for review + improvements on the wording.

In terms of DX / providing helpful feedback to the developer we can also consider logging something to the console (only when in dev-mode).